firebase-rules-coverage/package.json

```diff
@@ -25,17 +25,17 @@
     "release": "yarn clean && yarn compile && cp package.json ./build && cp README.md ./build && cp CHANGELOG.md ./build && yarn publish ./build"
   },
   "dependencies": {
-    "meow": "~12.1.1",
+    "meow": "~13.2.0",
     "source-map": "^0.7.4"
   },
   "devDependencies": {
     "@jest/globals": "^29.7.0",
     "@types/jest": "^29.5.12",
     "@types/node": "^16.18.97",
-    "gts": "^5.3.0",
+    "gts": "5.3.0",
     "jest": "^29.7.0",
     "ts-jest": "^29.1.4",
     "ts-node": "^10.9.2",
-    "typescript": "~5.1.6"
+    "typescript": "~5.5.4"
```

Comment:
@alexregier Have you tried <=5.4 version?

Reply:
see comment: https://github.com/simpleclub/firebase-rules-helper/pull/29#issuecomment-2315405180
5.1 would probably work, but 5.5 isn't the issue